lib/Service/LangRopeService.php

```diff
@@ -27,6 +27,11 @@
 use Psr\Log\LoggerInterface;
 use RuntimeException;
 
+enum ScopeType: string {
```

Comment:
Nextcloud 29 still supports php 8.0, which [doesn't have enums](https://www.php.net/manual/en/language.types.enumerations.php), sadly

Reply:
oh no. Will update it to use something else and also add a case in the workflow matrix for that. Thanks!